packages/codefixes/test/__snapshots__/base-codefixes.test.ts.snap

```diff
@@ -563,6 +563,39 @@ export function think(
 export function thinkTwice(a) {
   return a;
 }
+
+/**
+ * The return type is throwing an error
+ * @param {[number, number]} a-and-b
+ * @param {Array<string>} c
+ * @return {Array<{content: string}>}
+ */
+// @ts-expect-error @rehearsal TODO TS7031: Binding element 'a' implicitly has an 'any' type.
```

Comment:
This works in the next function, but not here.

Created a ticket for this [case](https://github.com/rehearsal-js/rehearsal-js/issues/1062)